Core/TestsFoundation/Helpers/OfflineHelper.swift

```diff
@@ -0,0 +1,34 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+public class OfflineHelper: BaseHelper {
+    
+    // MARK: UI Elements
+    public static var offlineModeAlert: XCUIElement { app.find(label: "Offline mode", type: .alert) }
+    public static var offlineModeAlertTitleText: XCUIElement { offlineModeAlert.find(label: "Offline mode", type: .staticText) }
+    public static var offlineModeAlertMessageText: XCUIElement { offlineModeAlert.find(label: "This item is not available offline.", type: .staticText) }
+    public static var offlineModeAlertOkButton: XCUIElement { offlineModeAlert.find(label: "OK", type: .button) }
+
+
+    public static func assertOfflineAlertDialogContent() {
+        XCTAssertTrue(OfflineHelper.offlineModeAlert.isVisible)
+        XCTAssertTrue(OfflineHelper.offlineModeAlertTitleText.isVisible && OfflineHelper.offlineModeAlertTitleText.hasLabel(label: "Offline mode"))
+        XCTAssertTrue(OfflineHelper.offlineModeAlertMessageText.isVisible && OfflineHelper.offlineModeAlertMessageText.hasLabel(label: "This item is not available offline."))
+        XCTAssertTrue(offlineModeAlertOkButton.isVisible && offlineModeAlertOkButton.isHittable)
+    }
```

Comment:
We don't put assertions in helper classes. Please only put assertions inside an actual test case to be consistent with all the other tests.

Reply:
Ok, I've put the assertions in the test case. I just wanted to avoid code duplication because I used these same assertion flow multiple times but I can adopt to your conventions, thanks for the feedback. :)